enforcer-parent/enforcer/src/main/resources/Dockerfile.ubuntu

```
@@ -56,6 +56,7 @@ RUN \
     groupadd --system -g ${MG_USER_GROUP_ID} ${MG_USER_GROUP} \
     && useradd --system --create-home --home-dir ${MG_USER_HOME} --no-log-init -g ${MG_USER_GROUP_ID} -u ${MG_USER_ID} ${MG_USER} \
     && mkdir ${MG_USER_HOME}/logs && chown -R ${MG_USER}:${MG_USER_GROUP} ${MG_USER_HOME}/logs  \
+    && mkdir -p ${MG_USER_HOME}/lib/dropins && chown -R ${MG_USER_HOME}/lib/dropins \
```

Shall we do the same way we have done in `enforcer-parent/enforcer/src/main/resources/Dockerfile`
```suggestion
    && mkdir -p ${MG_USER_HOME}/lib/dropins && mkdir -p ${MG_USER_HOME}/logs \
    && chown ${MG_USER}:${MG_USER_GROUP} -R ${MG_USER_HOME} \
```

---

fixed, changed all Dockerfiles to `mkdir` the required paths first and then `chown` the `MG_USER_HOME`